README.md

```diff
@@ -1149,13 +1244,23 @@ The following service template can be used and modified as needed:
  
 `environments` Service environments. Object where each first-level property is the name of a service environment. Optional.
  
- `environments.[envName].secret` Shared secret to use when communicating with the service.
+ `environments.[envName].secret` Shared secret to use when communicating with the service. Аpplicable and required for 'flex-internal' and 'flex-external'
  
-`environments.[envName].host` URI pointing to the service server's location. Required when `type` is 'flex-external'.
+`environments.[envName].host` URI pointing to the service server's location. Required for all service types except 'flex-internal' and 'rapid-health'.
 
 `environments.[envName].sourcePath` Path to source code - relative or absolute. Optional. Applicable when `type` is 'flex-internal'.
 
-`environments.[envName].environmentVariables` Environment variables. Object. Optional.
+`environments.[envName].runtime` The nodejs version of the runtime. Optional. The value must be one of [node6, node8, node10, node12]. Applicable when `type` is 'flex-internal'.
+
+`environments.[envName].authentication` Authentication type and credentials. Object. Optional. Applicable for RapidData services
```

Comment:
I think that instead of RapidData we should specify the types as they are in the CLI (e.g. `rapid-health`). Otherwise, it's a bit confusing.